content/articles/testing-for-broken-authentication-in-web-apps/index.md

```diff
@@ -0,0 +1,117 @@
+Authentication is a mechanism put in place to determine if a user is who they say they are, through either a username and password-based system or any other form of authentication system.
+
```

Comment:
A glimpse of what is going to covered in this article - would help.

Specify the goals and objectives.

Also, add a `Table of contents` and `Pre-requisites` sections.